dap_prinz_green_jobs/pipeline/green_measures/industries/README.md

```diff
@@ -1,28 +1,30 @@
-# Industry Green Measures
+# 🏭🥬 Industry Green Measures
 
-## Industry data processing
+This directory is organised as follows:
 
-As a one-off run:
+1. `sic_mapper/`: The directory that maps job adverts to SIC codes.
+2. `prodigy/`: The directory to generate training data to train a company description classifier for the `SicMapper`.
+3. `evaluation/`: The directory to evaluate the `SicMapper` class.
 
-```
-python dap_prinz_green_jobs/pipeline/green_measures/industries/industries_data_processing.py
-```
+It also contains the following `.py` files:
 
-This will save out just the key columns from the full Companies House dataset to `s3://prinz-green-jobs/inputs/data/industry_data/BasicCompanyDataAsOneFile-2023-05-01_key_columns_only.csv`. This speeds up the rest of the pipeline as loading the full dataset is slow. NOTE: this script will take a long time to run.
+1. `industries_data_processing.py`: A script to process the Companies House dataset.
```

Comment:
is everything to do with companies house just for the evaluation now? perhaps worth saying here if so, or even moving this script to the evaluation folder?

Reply:
yeah i guess this isn't super relevant anymore - not sure quite what to do with this script